### PR TITLE
Cory/inf 627 address validation web sdk

### PR DESCRIPTION
**Pull request by corypisano**
- Add `radar.ValidateAddress()` method 
- Add `expandUnits` as a param to `radar.autocomplete()`

QA: 

validateAddress() response:
Screen Shot 2023-03-09 at 5 46 42 PM

### CODE REVIEW

File: test/api/addresses.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  const country = 'US';
+  const state = 'NY';
+  const city = 'New York';
+  const number = '841';
+  const postalCode = '10010';
+  const street = 'Broadway';
+  const unit = '7';
+
+  const autocompleteResponse = { meta: {}, address: {} };
```
Comment (bschoeneweis) [inline test/api/addresses.test.js:23]: nit: `validateResponse`

**Review by bschoeneweis — APPROVED** [review]
Looks good 👍

File: src/api/addresses.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Http from '../http';
+
+class Addresses {
+  static async validateAddress(addressOptions={}) {
+    let {
```
Comment (kochis) [inline src/api/addresses.js:5]: should this be `const`

Comment (corypisano) [reply src/api/addresses.js:5]: ye

**Review by tjulien — DISMISSED** [review]

File: src/api/addresses.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import Http from '../http';
+
+class Addresses {
+  static async validateAddress(addressOptions={}) {
+    const {
+      country,
+      state,
```
Comment (bschoeneweis) [inline src/api/addresses.js:7]: changed to `countryCode` and `stateCode`

**Review by bschoeneweis — CHANGES_REQUESTED** [review]

File: src/api/search.js [hunk 4]
```diff
@@ -89,6 +89,7 @@ class Search {
       limit,
       layers,
       country,
```
Comment (bschoeneweis) [inline src/api/search.js:91]: We can also update to `countryCode` here (will accept in the endpoint now)

Comment (corypisano) [reply src/api/search.js:91]: ok yea i guess we can support both so nothing breaks

**Review by bschoeneweis — DISMISSED** [review]
Cool will leave to you whether or not to update `/search/autocomplete`, probably good to follow suite w/ iOS and android

**Comment by corypisano** [comment]
image

File: src/api/addresses.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+class Addresses {
+  static async validateAddress(addressOptions={}) {
+    const {
+      countryCode,
+      stateCode,
+      city,
+      number,
+      postalCode,
+      street,
+      unit // optional
```
Comment (bschoeneweis) [inline src/api/addresses.js:12]: also will want to add `addressLabel`.  not sure if we care to do any validation in the sdk for param combos or leave it to the api

**Review by lmeier — APPROVED** [review]
👍

**Pull request merged**